models/base/BCEfficiencyFitter.h

```diff
@@ -64,28 +75,17 @@ class BCEfficiencyFitter : public BCFitter
     /* @{ */
 
     /**
-     * The default constructor. */
-    BCEfficiencyFitter();
```

Comment:
because the defaulting of the name in the name-only constructor covers this. this matches other code in the models/base directory